src/main/scala/where/Predicate.scala

```diff
@@ -1,10 +1,32 @@
 package com.originate.scalypher.where
 
 import com.originate.scalypher.ConstantString
+import com.originate.scalypher.path.Path
 import com.originate.scalypher.ToQuery
 
-sealed trait Predicate extends ToQuery
+case class PredicateWithProjection(
+  predicate: Predicate,
+  projection: PathProjection
+) {
+  def where(f: ObjectReference => Where): PredicateCondition =
+    PredicateCondition(predicate, projection, f)
+}
+
+sealed trait Predicate extends ToQuery {
+  def nodesIn(path: Path): PredicateWithProjection =
+    PredicateWithProjection(this, Nodes(path))
+
+  def nodeIn(path: Path): PredicateWithProjection =
+    PredicateWithProjection(this, Nodes(path))
```

Comment:
see comment above, this is equivalent to a testing framework aliasing "an" and "a" like `an [Exception] ...` or `a [NullPointerException`...